kasa/smart/smartdevice.py

```diff
@@ -215,15 +265,19 @@ async def _query_helper(
         return res
 
     @property
-    def state_information(self) -> Dict[str, Any]:
-        """Return the key state information."""
+    def ssid(self) -> str:
```

Comment:
Would we still have `ssid` as a public property or would it be deprecated?

Reply:
This was never really exposed earlier, so we can skip adding it by using a getter that accesses the internals directly. Maybe that's indeed a better approach, instead of extending the public API.

Reply:
I think as long as it's not on the base `Device` it's probably fine as is